pkg/bpf/setup_linux.go

```diff
@@ -16,11 +16,11 @@ import (
 
 func __mount() error {
 	// Check if the path exists.
-	_, err := os.Stat(plugincommon.FilterMapPath)
+	_, err := os.Stat(plugincommon.MapPath)
 	if err != nil {
 		if os.IsNotExist(err) {
 			// Path does not exist. Create it.
-			err = os.MkdirAll(plugincommon.FilterMapPath, 0o755)
+			err = os.MkdirAll(plugincommon.MapPath, 0o755)
```

Comment:
I'd say just suppress that linter. Seeing octal 755 in anything involving directories is self-explanatory.